docs/source/events.md

```diff
@@ -138,15 +138,16 @@ indication on event ordering *within each individual component*.
     cmd                 : command received from pmgr                 (uid: pilot, msg: command, [API])
     get                 : tasks   received from task manager         (uid: pilot, msg: 'bulk size: %d')
     get                 : task    received from task manager         (uid: task)
-    dvm_start           : DVM startup by launch method               (uid: pilot) [CFG-R])
-    dvm_ok              : DVM startup completed                      (uid: pilot) [CFG-R])
-    dvm_fail            : DVM startup failed                         (uid: pilot) [CFG-R])
-    dvm_stop            : DVM stopped                                (uid: pilot) [CFG-R])
+    dvm_start           : DVM startup by launch method               (uid: pilot, msg: 'dvm_id=%d') [CFG-DVM])
+    dvm_uri_ok          : DVM URI is set successfully                (uid: pilot, msg: 'dvm_id=%d') [CFG-DVM])
```

Comment:
I can see that `dvm_ok` does not match - but maybe shorter as `dvm_uri`?

Reply:
ok, will update accordingly